don't go adding emtpy -I/-L flags.  this has been fixed upstream.

http://bugs.gentoo.org/152408

--- configure.in
+++ configure.in
@@ -10207,10 +10207,10 @@
 
 dnl Check for AC_PATH_X variables
-if test "X$x_includes" != "XNONE"; then
+if test "X$x_includes" != "XNONE" -a "X$x_includes" != "X"; then
 	CFLAGS="$CFLAGS -I$x_includes"
 	CXXFLAGS="$CXXFLAGS -I$x_includes"
 fi
-if test "X$x_libraries" != "XNONE"; then
+if test "X$x_libraries" != "XNONE" -a "X$x_libraries" != "X"; then
 	CFLAGS="$CFLAGS -L$x_libraries"
 	CXXFLAGS="$CXXFLAGS -L$x_libraries"
 fi
